### PR TITLE
drpolicy.yaml: drClusterSet and schedulingInterval values specify

### PR DESCRIPTION
**Pull request by hatfieldbrian**
Signed-off-by: hatfieldbrian <bhatfiel@redhat.com>

### CODE REVIEW

**Comment by hatfieldbrian** [comment]
This pull request reflects the following `DRPolicy` field additions:

- `drClusterSet` https://github.com/RamenDR/ramen/pull/117
- `schedulingInterval` https://github.com/RamenDR/ramen/pull/182

File: subscriptions/drpolicy.yaml [hunk 1]
```diff
@@ -4,6 +4,9 @@ kind: DRPolicy
 metadata:
   name: dr-policy
 spec:
-  clusterNames:
-  - hub
-  - cluster1
+  drClusterSet:
+  - name: hub
+    s3ProfileName: ""
```
Comment (vdeenadh) [inline subscriptions/drpolicy.yaml:9]: I suggest setting the s3ProfileName to minio-on-hub to match the e2e setup, like [here](https://github.com/vdeenadh/ocm-ramen-samples/blob/s3/subscriptions/drpolicy.yaml).

Comment (hatfieldbrian) [reply subscriptions/drpolicy.yaml]: Updated.  Thank you Veera!

**Comment by vdeenadh** [comment]
LGTM

**Review by ShyamsundarR — APPROVED** [review]

**Pull request merged**